mix.exs

```diff
@@ -224,7 +224,7 @@ defmodule Logflare.Mixfile do
       "lint.diff": ["credo diff master"],
       "lint.all": ["credo --strict"],
       "ecto.seed": ["run priv/repo/seeds.exs"],
-      "ecto.setup": ["ecto.create", "ecto.migrate"],
+      "ecto.setup": ["ecto.create", "ecto.migrate", "ecto.seed"],
```

Comment:
this would seed twice when running `mix setup` for setting up the dev environment.

I was referring to moving the initial repo inserts that you had written to the seed.exs so that we can remove the raw inserts.